MAUI/Rating/Getting-Started.md

```diff
@@ -187,4 +187,6 @@ public MainPage()
 
 ![SfRating Getting Started ](images/getting-started.png)
 
-The complete Getting Started sample is available in this [link.](https://github.com/SyncfusionExamples/maui-rating-samples)
+N>
+* Get the complete getting started sample from [GitHub](https://github.com/SyncfusionExamples/maui-rating-samples) link.
+* You can refer to our [.NET MAUI Rating](https://www.syncfusion.com/maui-controls/maui-rating) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Rating Example](https://github.com/syncfusion/maui-demos/tree/master) that shows you how to render the Rating in .NET MAUI.
```

Comment:
Recheck with PO regarding the correct demo link for rating

The below is fine...no need to delete/modify GS sample link.
N> You can refer to our [.NET MAUI Rating](https://www.syncfusion.com/maui-controls/maui-rating) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI Rating Example](https://github.com/syncfusion/maui-demos/tree/master) that shows you how to render the Rating in .NET MAUI.

Reply:
Okay